### PR TITLE
#315 - GraphQL documentation correction

### PR DESCRIPTION
**Pull request by vladyslav-fenchak**

### CODE REVIEW

File: packages/plugins/minos-router-graphql/minos/plugins/graphql/handlers.py [hunk 1]
```diff
@@ -60,6 +66,10 @@ def _build_response_from_graphql(result: ExecutionResult) -> Response:
             for error in errors:
                 if isinstance(error.original_error, ResponseException):
                     status = error.original_error.status
+                    error_trace = "".join(traceback.format_tb(error.original_error.__traceback__))
+                    logger.exception(f"Raised a system exception:\n {error_trace}")
+                else:
+                    logger.error(f"Raised an application exception:\n {error}")
```
Comment (garciparedes) [inline packages/plugins/minos-router-graphql/minos/plugins/graphql/handlers.py:72]: Hi @vladyslav-fenchak! I think you reversed the log messages. In any case, I would use the error_trace in both cases 🙂

**Review by garciparedes — CHANGES_REQUESTED** [review]
Hi @vladyslav-fenchak, it looks good to me, but before merging could you resolve the comment I left to you and also check the case in which a query and a mutation has the same name?

File: packages/plugins/minos-router-graphql/minos/plugins/graphql/handlers.py (hunk 1 above)
Comment (vladyslav-fenchak) [reply packages/plugins/minos-router-graphql/minos/plugins/graphql/handlers.py]: Done!

**Review by garciparedes — APPROVED** [review]
Hi @vladyslav-fenchak, it LGTM!

**Pull request merged**